pkgs/top-level/all-packages.nix

```
@@ -12944,7 +12944,7 @@ in
 
   herqq = libsForQt514.callPackage ../development/libraries/herqq { };
 
-  hidapi = callPackage ../development/libraries/hidapi { };
+  hidapi = callPackage ../development/libraries/hidapi { systemd = systemdMinimal; };
```

I just checked the source code, and `hidapi` and `libusb1` actually depends on `udev`; not `systemd` (though of course those are one and the same package).   For documentation sake it would be nice to name the parameters `udev` instead of `systemd`; and then you don't need to repeat `systemdMinimal` all over the place but just need the toplevel `udev = systemdMinimal` alias.

If that's something you're quickly willing to do; I think it would be a lot clearer for the next reader. But also feel free to ignore this as bikeshedding.  It's not a blocker by any means

---

Pointing `udev` to `systemdMinimal` would be a logical consequence, but this might cause quite some rebuilds and possibly break other stuff, as it hasn't been used properly/consistently throughout nixpkgs. This should definitely be another PR.

---

Another PR is fine --- i doubt these changes are necessary for this one at all, but I think we can still rename the variable before the big change, giving us:
```suggestion
  hidapi = callPackage ../development/libraries/hidapi { udev = systemdMinimal; };
```
and separately worry about making `udev = systemdMinimal` everywhere.

---

My suggestion is now done for both those packages.